modules/metastorage-api/src/main/java/org/apache/ignite/internal/metastorage/MetaStorageManager.java

```diff
@@ -60,6 +61,13 @@ public interface MetaStorageManager extends IgniteComponent {
      */
     CompletableFuture<Entry> get(ByteArray key, long revUpperBound);
 
+    /**
+     * Retrieves entries for the given key and bound revisions.
+     * TODO: IGNITE-19735 move this method to another interface for interaction with local KeyValueStorage.
+     */
+    @Deprecated
+    List<Entry> getEntriesLocally(byte[] key, long revLowerBound, long revUpperBound);
```

Comment:
Let's be consistent with other methods naming. There are bunch of method in `MetaStorageManager`, `RocksDbKeyValueStorage` and other related places that return entries, however neither of them have entries in it's name, you on the contrary introduces getEntriesLocally. Please rename it and all related `getEntries`, `doGetEntries`, etc.

Reply:
Fixed. As we discussed I used a name `get` instead of `getEntries`